db/migrations/0159__add_tree_tooltip.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO options (name, value, utcDateCreated, utcDateModified, isSynced)
```

Comment:
DB upgrade is not necessary for options - it's enough it's specified in options\_init - it will be created if missing ...

Reply:
Thank you for the feedback @zadam 
Sure I will remove it 👍